editor/src/components/inspector/sections/layout-section/conditional-section.tsx

```diff
@@ -146,6 +150,15 @@ export const ConditionalSection = React.memo(({ paths }: { paths: ElementPath[]
     [conditionOverride, setConditionOverride],
   )
 
+  const replaceBranches = React.useCallback(
+    () => () => {
```

Comment:
I think you want to remove one of these functions. As `useCallback` will be memoising the function, but when it's assigned, the function is being invoked which will create a new function each time.